### DISCUSSION TITLE
Meta-Provblem: Quarto code on a Quarto slide

### DISCUSSION THREAD

**Discussion by kellertuer** [quarto-dev/quarto-cli#10062] [Q&A] [answered]
### Description

I would like to have a presentation with quarto code inside. I currently do on a slide

`````
````{markdown}
---
title: "A short example"
---

```{julia}
#| echo: false
#| code-fold: true
#| output: false
using Pkg;
cd(@__DIR__)
Pkg.activate("."); # for reproducibility use the local tutorial environment.
```

First, load all necessary packages

```{julia}
using Manopt, Manifolds
M = Sphere(2)
```
````
`````

And my problem is that though the `{Julia}` code cells are _within_ the markdown code cell, they are executed.

The overall quarto file (reveal js slides) has set

```
execute:
  eval: false
```

but that has no effect, also inside the markdown code cell this has no effect, also inside the Julia code cell this has no effect. Where else could I indicate I do not want to execute the code? 

Or: How can I get _exactly_ the displayed code here on a Reveal.jl slide in a Quarto file?

**Comment by mcanouil** [comment, marked as answer]
Using `{language}` means you want the content to be executed.

See ["escaping inline/code cell"](https://quarto.org/docs/computations/inline-code.html#escaping).

<table>
<tr><th>Quarto document</th><th>HTML</th></tr>
<tr><td>

`````qmd
---
format: html
---


````markdown
---
title: "A short example"
---

```{{julia}}
#| echo: false
#| code-fold: true
#| output: false
using Pkg;
cd(@__DIR__)
Pkg.activate("."); # for reproducibility use the local tutorial environment.
```

First, load all necessary packages

```{{julia}}
using Manopt, Manifolds
M = Sphere(2)
```
````

`````

</td><td>

<img width="644" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/1e3805f5-5fec-405e-9273-6a152959af28">

</td></tr>
</table>

**Reply by kellertuer** [reply]
Ah!
I was surprised that none of the `eval`s helped – and did not consider escaping because that was a block within a block – but this actually did help :) Thanks!
